tokio-io/Cargo.toml

```diff
@@ -21,3 +21,6 @@ categories = ["asynchronous"]
 bytes = "0.4.7"
 futures = "0.1.18"
 log = "0.4"
+
+[dev-dependencies]
+tokio-current-thread = "0.1.1"
```

Comment:
I think you need to specify a path here or it will pull it from crates.io:

```
tokio-current-thread = { version = "0.1.1", path = "../tokio-current-thread" }
```

Reply:
Ah right, good call.